README.md

```diff
@@ -206,11 +213,14 @@ that you have a 32bit version of Python installed.
 
 ### Set up your django environment
 
+Before running any of the commands below, you should have all of the following running: couchdb, redis, elasticsearch,
+and kafka. To install and run kafka, follow the instructions here: 
```

Comment:
True.  That's something we could make more explicit in the README - maybe split it up into "here's what you need for a bare-bones setup", "here's what you need for elasticsearch, celery, etcetera".
Also, just opened this PR for the `SHARED_DRIVE_ROOT` thing: https://github.com/dimagi/commcare-hq/pull/9934